AzureCalling/Views/ParticipantLabel.swift

```diff
@@ -9,13 +9,6 @@ import UIKit
 class ParticipantLabel: UILabel {
```

Comment:
Let's try to move those UI configuration into Storybroad configuration in ParticipantLabelView

Reply:
Moved the padding setup to ParticipantLabelView in Storyboard using `Layout Margins`